subset/switches/test_switch

```diff
@@ -8,8 +8,15 @@ if [ -n "$LOCAL_IP" ]; then
     ip addr add $LOCAL_IP dev `hostname`-eth0
     echo Switch test with port $SWITCH_PORT.
     ping -n -c 10 $SWITCH_IP
-    java -jar switches/target/switchtest-0.0.1-jar-with-dependencies.jar $SWITCH_IP $SWITCH_PORT
+    $POE_ENABLED=$(jq .modules.switch.poe.enabled /config/type/module_config.json)
```

Comment:
Also, this shouldn't be checking /config/type -- there shouldn't be a module_config.json file there at all... tests should ALWAYS test /config/device/module_config.json -- it's best follow established examples as much as possible unless there's a really good reason to deviate... In this case: https://github.com/faucetsdn/daq/blob/master/subset/pentests/test_nmap#L5

@InBrewJ , I'm guessing your setup didn't work because you were trying to put module_config.json files direclty in the aux/ directories -- you shouldn't do that, only the system should put them there (and in the one by mac address) -- take a look at [this documentation](https://github.com/faucetsdn/daq/blob/master/docs/site_path.md#module-configs) on how they are calculated.

Also, you don't need the "export" -- that makes it an environment variable which it doesn't need to be. In the end, it should look something like:

```
poe_enabled=$(jq .modules.switch.poe.enabled /config/device/module_config.json)
```

Reply:
I don't remember explicitly putting things in aux by hand - my setup wasn't working because `subset/switches/test_switch` (above) was looking in the /config/type directory, which didn't exist. Perhaps I misread (I'll confess I skim read to try and get to the bottom of the issue quickly) line 165 onwards of [host.py](https://github.com/grafnu/daq/blob/master/daq/host.py). I was under the impressed that the module_config.json can be written into the test docker container from either:

`{site_path}/device_types/{device_type}/aux/`
OR
`{site_path}/mac_addrs/{device_mac}/aux/`

- I made a modification to `subset/switches/test_switch` to look in `{site_path}/mac_addrs/{device_mac}/aux/` (where [host.py](https://github.com/grafnu/daq/blob/master/daq/host.py was copying the module_config.json from in my case) and then things started working

...but if things should only ever be in  /config/device/module_config.json, all the better! Thanks.